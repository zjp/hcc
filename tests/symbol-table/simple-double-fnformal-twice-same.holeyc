bool glob;
int b;
int main(int b){
	int a;
	glob = a;
}
int maintwo(int b){
	int a;
	glob = a;
}
