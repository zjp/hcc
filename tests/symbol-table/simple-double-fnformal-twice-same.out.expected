bool glob;
int b;
int main(int b){
	int a;
	glob(bool) = a(int);
}
int maintwo(int b){
	int a;
	glob(bool) = a(int);
}
